packages/cf-style-container/README.md

```diff
@@ -88,7 +88,7 @@ export default createComponentStyles({ mainStyles, legendStyles }, FormFieldset)
 Notice that rules are now an object. The names you chose will be used for classNames
 accessible as `styles.mainStyles` and `styles.legendStyles` in this case.
 
-### applyTheme(Component, theme)
+### applyTheme(Component, mainTheme, customTheme)
```

Comment:
Could we have even more flexible API using rest parameter syntax like

```js
applyTheme(Component, theme1, theme2 .... theme?)
```

similar to `Object.assign()`. It just seems more idiomatic than having two arguments called `main` and `custom`. What do you think?